src/js/bookmarklet.js

```diff
@@ -48,8 +48,10 @@ function iFrame(apiRoot) {
     }
 
     function grabCitation(){
-      var cite;
-      document.domain.split("\.")[0];
+      var cite = document.domain;
+      if (cite.indexOf('www.') == 0) {
+          cite = cite.slice(4);
+      }
```

Comment:
Nice, I didn't know about indexOf. I think we're still going to run into the problem we currently have on prod where document.domain returns "pullquote.knightlab.com", though.